src/Tribe/Importer/File_Importer_Venues.php

```diff
@@ -36,30 +35,162 @@ protected function create_post( array $record ) {
 		return $id;
 	}
 
+	/**
+	 * Build a venue array for creation/update of the current imported venue.
+	 *
+	 * @since 3.2
+	 * @since TBD Adjust to prevent overwriting values that aren't mapped.
+	 *
+	 * @param int    $venue_id The ID of the venue we're currently importing
+	 * @param array  $record An event record from the import.
+	 * @return array $venue The array of venue data for creation/update.
+	 */
 	private function build_venue_array( $venue_id, array $record ) {
-		$show_map_setting = tribe( 'events-aggregator.settings' )->default_map( 'csv' );
-		$venue_address    = trim( $this->get_value_by_key( $record, 'venue_address' ) . ' ' . $this->get_value_by_key( $record, 'venue_address2' ) );
-		$venue            = array(
-			'Venue'         => $this->get_value_by_key( $record, 'venue_name' ),
-			'Description'   => $this->get_value_by_key( $record, 'venue_description' ),
-			'Address'       => $venue_address,
-			'City'          => $this->get_value_by_key( $record, 'venue_city' ),
-			'Country'       => $this->get_value_by_key( $record, 'venue_country' ),
-			'Province'      => $this->get_value_by_key( $record, 'venue_state' ),
-			'State'         => $this->get_value_by_key( $record, 'venue_state' ),
-			'Zip'           => $this->get_value_by_key( $record, 'venue_zip' ),
-			'Phone'         => $this->get_value_by_key( $record, 'venue_phone' ),
-			'URL'           => $this->get_value_by_key( $record, 'venue_url' ),
-			'ShowMap'       => $venue_id ? get_post_meta( $venue_id, '_VenueShowMap', true ) : $show_map_setting,
-			'ShowMapLink'   => $venue_id ? get_post_meta( $venue_id, '_VenueShowMapLink', true ) : $show_map_setting,
-			'FeaturedImage' => $this->get_featured_image( $venue_id, $record ),
+		$venue          = [];
+		$columns        = [
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 1 space but found 8 spaces

<sub>hash: 964787f4b1d82eea102b872a83d823ff</sub>